testsuite/features/support/api_test.rb

```diff
@@ -17,6 +17,11 @@
 
 # Abstract parent class describing an API test
 class ApiTest
+  ##
+  # It creates a bunch of objects that are used to interact with the API
```

Comment:
```suggestion
  # It creates objects that are used to interact with the API
```
Writing "a bunch" doesn't really make it more descriptive and doesn't feel appropriate here